src/test/browser/browser-common.js

```diff
@@ -111,6 +114,9 @@ afterEach(async () => {
 });
 
 async function saveStepScreen(stepStr) {
+  if (process.env.DISABLE_SCREENSHOT_COMPARISON) {
```

Comment:
I think this check can be removed so that we can still see the output screenshots even if we don't want to compare them.